tests/e2e/cluster_downgrade_test.go

```diff
@@ -62,36 +69,50 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 
 	t.Log("Downgrade enabled, validating if cluster is ready for downgrade")
 	for i := 0; i < len(epc.Procs); i++ {
-		e2e.AssertProcessLogs(t, epc.Procs[i], "The server is ready to downgrade")
```

Comment:
Would like to leave checking this log. Can you clarify why you removed it?

Reply:
Please check this reply https://github.com/etcd-io/etcd/pull/14657#issuecomment-1296493139 Thanks!

Reply:
Added it back. PTAL